.github/workflows/workflow_branch.yml

```
@@ -1,115 +1,107 @@
-        DB_NAME: photo_market
-        DB_HOST: 127.0.0.1
-        DB_PORT: 5432
-      run: |
-        python -m flake8 .
-
-  build_and_push_to_docker_hub:
-    name: Push backend Docker image to DockerHub
-    runs-on: ubuntu-latest
-    needs: tests
-    steps:
-      - name: Check out the repo
-        uses: actions/checkout@v3
-      - name: Set up Docker Buildx
-        uses: docker/setup-buildx-action@v2
-      - name: Login to Docker
-        uses: docker/login-action@v2
-        with:
-          username: ${{ secrets.DOCKER_USERNAME }}
-          password: ${{ secrets.DOCKER_PASSWORD }}
-      - name: Push to DockerHub
-        uses: docker/build-push-action@v4
-        with:
-          context: .
-          push: true
-          tags: 4aiz/photo_market_backend:latest
-
-  build_gateway_and_push_to_docker_hub:
-    name: Push gateway Docker image to DockerHub
-    runs-on: ubuntu-latest
-    steps:
-      - name: Check out the repo
-        uses: actions/checkout@v3
-      - name: Set up Docker Buildx
-        uses: docker/setup-buildx-action@v2
-      - name: Login to Docker
-        uses: docker/login-action@v2
-        with:
-          username: ${{ secrets.DOCKER_USERNAME }}
-          password: ${{ secrets.DOCKER_PASSWORD }}
-      - name: Push to DockerHub
-        uses: docker/build-push-action@v4
-        with:
-          context: ./nginx/
-          push: true
-          tags: 4aiz/photo_market_gateway:latest
-
-  deploy:
-    runs-on: ubuntu-latest
-    needs:
-      - build_and_push_to_docker_hub
-      - build_gateway_and_push_to_docker_hub
-    steps:
-    - name: Checkout repo
-      uses: actions/checkout@v3
-    - name: Copy docker-compose.yml via ssh
-      uses: appleboy/scp-action@master
-      with:
-        host: ${{ secrets.HOST }}
-        username: ${{ secrets.USER }}
-        key: ${{ secrets.SSH_KEY }}
-        passphrase: ${{ secrets.SSH_PASSPHRASE }}
-        source: "docker-compose.production.yml"
-        target: "photo_market"
-    - name: Executing remote ssh commands to deploy
-      uses: appleboy/ssh-action@master
-      with:
-        host: ${{ secrets.HOST }}
-        username: ${{ secrets.USER }}
-        key: ${{ secrets.SSH_KEY }}
-        passphrase: ${{ secrets.SSH_PASSPHRASE }}
-        script: |
-          cd photo_market
-          sudo docker compose -f docker-compose.production.yml pull
-          sudo docker compose -f docker-compose.production.yml down
-          sudo docker compose -f docker-compose.production.yml up -d
-          sudo docker compose -f docker-compose.production.yml exec backend python manage.py migrate
-          sudo docker compose -f docker-compose.production.yml exec backend python manage.py collectstatic
-          sudo docker compose -f docker-compose.production.yml exec backend cp -r /app/static/. /backend_static/static/
\ No newline at end of file
+name: Photo market workflows
+
+on:
+  push:
+    branches:
+      - feature/chat_ws

```

Но мы же не будем в эту ветку больше пушить после PR, так ведь? Да вообще я считаю, что должно быть 
```suggestion
on:
  pull_request:
    branches:
      - develop
    types: [closed]

```
Закрыли PR смержили в основную ветку - сразу идем деплоить. А закрытый PR хотя бы предполагает, что код кто-то проверял и косяков меньше будет.

---

Главное что бы PR проверяли быстрей чем за 3 дня ))